sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Hook/HookInfoPatch.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    /// <summary> The HookInfoPatch. </summary>
+    internal partial class HookInfoPatch
```

Comment:
Info for Information? in general we shouldn't use abbreviations in our names

Reply:
Do you feel it is worth expanding the generated name even if it is internal?

Reply:
I would personally would prefer to so that we are consistent. If we have other internal classes that are like this, then it might not be worth changing

Reply:
Thanks for the suggestion - I'm going to punt this for now.